solutions/standard/variables.tf

```diff
@@ -87,7 +87,7 @@ variable "public_engine_enabled" {
 variable "public_engine_name" {
   type        = string
   description = "The name of the IAM engine used to configure a Secrets Manager public certificate engine for an existing instance."
-  default     = "public-engine-sm"
+  default     = "public-engine-secrets-manager"
```

Comment:
Perhaps go with `public-cert-engine`. I think the variable should also be renamed to `public_cert_engine_name`